packages/jss/src/utils/createGenerateId.js

```diff
@@ -1,27 +1,26 @@
 /* @flow */
-import warning from 'warning'
+import warning from 'tiny-warning'
 import type {Rule, GenerateId} from '../types'
 import StyleSheet from '../StyleSheet'
 import moduleId from './moduleId'
 
 const maxRules = 1e10
 
-const env = process.env.NODE_ENV
-
 /**
  * Returns a function which generates unique class names based on counters.
  * When new generator function is created, rule counter is reseted.
  * We need to reset the rule counter for SSR for each request.
  */
 export default (): GenerateId => {
   let ruleCounter = 0
+  const env = process.env.NODE_ENV
```

Comment:
We cache it here so that we can still manipulate it in tests and don't have a negative impact on performance on when doing SSR.